### ISSUE TITLE
Use updated Application CRD ( apiextensions/v1beta1 to v1 )

### ISSUE THREAD

**Issue by sbose78** [argoproj-labs/applicationset#145]
after https://github.com/argoproj/argo-cd/pull/5516 is merged. 

Creating this issue as an FYI though I wouldn't expect any change in how the Golang dependency is consumed.

**Comment by jgwest** [comment]
Thanks @sbose78, ApplicationSet controller switched to non-beta v1 version with https://github.com/argoproj-labs/applicationset/pull/128, and I've confirmed that Argo CD's switch does not affect our functionality. 

I've swapped the integrated [Argo CD + ApplicationSet controller install YAML](https://github.com/argoproj-labs/applicationset/blob/release-0.1.0-rc/manifests/install-with-argo-cd.yaml) to Argo CD's master branch (eg what will become v1.9.0, as there isn't a v1.9 tag/branch yet), and everything works as expected there. (It will officially swap on release of ApplicationSet controller v0.1.0.)
		
So we should be good to go, absent any additional issues found during testing.
